info2439/module7/assignment6.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Camden Grove | INFO 2439 | Assignment3</title>
+  <meta charset="utf-8"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+ 
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+ 
+  
+  <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+     <link rel="apple-touch-icon" href="fish-icon_1_180x180.png">
+  <link rel="apple-touch-icon" href="fish-icon_2_152x152.png">
+  <link rel="apple-touch-icon" href="fish-icon_3_120x120.png">
+  <link rel="apple-touch-icon" href="fish-icon_4_76x76.png">
+  <link rel="apple-touch-icon" href="fish-icon_4_76x76.png">
+  <link rel="apple-touch-icon" href="fish-icon_192x192.png">
```

Comment:
great apple icons! I would have liked to see you add an icon for the page to display on the tab. checkout <icon> for that functionality.